packages/contracts/contracts/L1/IValidatorManager.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+/**
+ * @title IValidatorManager
+ * @notice Interface for ValidatorManager contract.
+ */
+interface IValidatorManager {
+    /**
+     * @notice Emitted when registers as a validator.
+     *
+     * @param validator               Address of the validator.
+     * @param started                 If the validator is started or not.
+     * @param commissionRate          The commission rate the validator sets.
+     * @param commissionMaxChangeRate The max change rate of the commission the validator sets.
+     * @param assets                  The number of assets the validator self-delegates.
+     */
+    event ValidatorRegistered(
+        address indexed validator,
+        bool indexed started,
```

Comment:
For filtering only the started validator, because if a validator starts with registration, the validator does not emit started event. But maybe I think it would be unnecessary.

Reply:
Yep I don't think it can filter all started validator, so maybe it's unnecessary.